app/dispatches/players/enlist.rb

```diff
@@ -15,13 +13,13 @@ def initialize(game:, player:)
 
     def assign_game_to_player!
       @player = Player.create!(nickname: nickname, game: game, life: Player::LIFE)
-      @game.update(state: 'running')
+      game.update(state: 'running')
```

Comment:
Why not for the methods changing the state. Is it too much to put them in the same loop as the predicate methods you have on your branch?